server.js

```diff
@@ -45,19 +45,16 @@ function initServer(member, alias) {
 
     app.get('/redeem', urlencodedParser, function (req, res) {
         //get the token ID from the callback url
-        Token.parseTokenRequestCallbackUrl(req.protocol + '://' + req.get('host') + req.originalUrl)
-            .then(function(tokenId) {
-                console.log(tokenId.tokenId);
-                member.getToken(tokenId.tokenId)
-                    .then(function (token) {
-                        // Redeem the token to move the funds
-                        member.redeemToken(token, 4.99, 'EUR', 'Book Purchase')
-                            .then(function (transfer) {
-                                console.log('\n Reedeem Token Response:', transfer);
-                                res.status(200);
-                                res.json(transfer)
-                            });
-                    })
+        var tokenId = req.query.tokenId;
```

Comment:
why did you take out parse?

Reply:
because it doesn't work with the popup - unless this has been fixed?

Reply:
https://tokenio.atlassian.net/browse/RD-747